roles/identity/tasks/realm.yml

```diff
@@ -0,0 +1,41 @@
+- name: Fetch Alfresco theme
+  become: true
+  vars:
+    alfresco_theme_version: 0.3.5
```

Comment:
please make it a role arg

Reply:
I'd prefer not since is not something that the average role user want to explicitly change